src/styles/page-template/index.md.njk

```diff
@@ -115,6 +122,13 @@ Variables can be set with:
       <td class="govuk-table__cell">Add a class to the <code>&lt;main&gt;</code> element. </td>
     </tr>
 
+    <tr class="govuk-table__row">
+      <td class="govuk-table__cell">mainLang</td>
+      <td class="govuk-table__cell">
+        Set the language of the <code>&lt;main&gt;</code> element. This is useful if some of the page elements, such as navigation and footer, are in English whereas the title and page content are in a different language. It is not necessary to set this if it matches <code>htmlLang</code>.
```

Comment:
Similar suggestion here as the one above:

```suggestion
        Set the language of the page content if it's different to <code>htmlLang</code>.
```

Reply:
New suggestion:
Set the language of the `<main>` element if it's different to <code>htmlLang</code>.